src/common/config_opts.h

```diff
@@ -13,7 +13,7 @@
  */
 
 /* note: no header guard */
-OPTION(host, OPT_STR, "localhost")
+OPTION(host, OPT_STR, "default")  // "default" means that host will be `hostname -s` 
```

Comment:
if we think it is important, than we should make "" the value that looks at the real hostname.  meaning the config option is nothing more than an override